dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java

```diff
@@ -2175,14 +1885,6 @@ public void refreshTagsPopUp(FileMetadata fm) {
 
     private List<String> tabFileTagsByName;
```

Comment:
If the tabFileTagsByName is used then getters and setters cannot be removed. The jsf under the hoods uses setters and getters to access/set value in xhtml, even if in the xhtml only field is referenced. 
Make sure to check that.

Reply:
Seems like I've have wrongly expected IntelliJs 'smart delete' with "search for text occurrences" to work better. Got to recheck all fields manually again.

Reply:
It's possible that IJ also recognizes accessing the field directly in xhtml as using getter, Im just just not sure for 100%.

Reply:
Looks like id don't. For example if you have a field seemingly unused and u perform smart delete on it, IJ will tell ask you if you want to delete getters and setters associated with it, since he sees them as unused aswell. Hence the problem.